test/models/patient_test.rb

```diff
@@ -78,6 +81,31 @@ class PatientTest < ActiveSupport::TestCase
     end
   end
 
+  describe 'pledge_summary' do
+    it "should return sent and remaining pledge for the next 7 days" do
+      # Doesn't error when there are no pregnancies
```

Comment:
Since everything gets rebuilt from scratch in every it block, you can delete anything within a test and it'll be rebuilt for the next test. Try Patient.destroy_all at the top of this particular test -- that'll run a command equivalent to delete from patients (or whatever the mongodb equivalent is, I forget exactly).

ps let me know if this isn't helpful or if it's making it too overly complicated -- this is pretty much good to and we're well into optimizing territory! 💯